plugin.video.intergalacticfm/README.md

```diff
@@ -0,0 +1,76 @@
+# Intergalactic FM TV video plugin for Kodi
+
+![Intergalactic FM TV icon](resources/icon.png?raw=true)
+
+
+## Installation
+
+This add-on can be installed directly from Kodi's
+main repository and it will update itself. For more information, see
+http://kodi.wiki/view/Add-on:Intergalactic_FM_TV
+
+Stepts to take to install from the Kodi main repository are:
+1. Go to **Add-ons** in the main menu
+2. Go to the **Add-on browser** (the box icon at the top left of the screen)
+3. Choose **Install from repository**
+4. Choose **Video add-ons**
+5. Navigate to **Intergalactic FM TV** and select it
+6. Choose **Install**
+
+### Linux
+
+Install Kodi with
+
+    sudo apt-get install kodi kodi-repository-kodi
```

Comment:
`kodi-repository-kodi` is only required if you want to use the broken Debian package. Remove it or mention it only in a note.